learning_journal/models/mymodel.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import (
+    Column,
+    Index,
+    Integer,
+    UnicodeText,
+    Unicode,
+    Date
+)
+
+from .meta import Base
+from datetime import datetime
+
+
+def _now():
+    return datetime.now()
+
+
+class Entry(Base):
+    __tablename__ = 'entries'
+    id = Column(Integer, primary_key=True)
+    title = Column(Unicode)
+    creation_date = Column(Date, default=_now)
```

Comment:
good find!  You can also pass the `datetime.datetime.now` method itself, instead of wrapping it in a new callable:

```python
creation_date = Column(Date, default=_now)
```